app/src/js/components/Footer/footer.js

```diff
@@ -18,7 +20,10 @@ class Footer extends React.Component {
       <div className='footer'>
         <div className='api__summary'>
           { authenticated &&
-            <h5 className='api__version'>Cumulus API Version: { versionNumber }</h5>
+            <div>
+              <h5 className='dashboard_version'> Cumulus Dashboard Version: { DASH_VERSION } </h5>
+              <h5 className='api__version'> Cumulus API Version: { versionNumber } </h5>
```

Comment:
Looks like a test is failing due to the spacing around this
```suggestion
              <h5 className='api__version'>Cumulus API Version: { versionNumber }</h5>
```

Also, while we're at it, can we add a similar test for Dashboard version?